apps/system/test/marionette/lib/faketextselectionapp.js

```diff
@@ -14,13 +14,21 @@ function FakeTextSelectionApp(client) {
 module.exports = FakeTextSelectionApp;
 
 FakeTextSelectionApp.Selector = Object.freeze({
-  bottomLeftInput: '#bottom-left-input',
-  bottomRightInput: '#bottom-right-input',
-  centerInput: '#center-input',
-  topLeftInput: '#top-left-input',
-  topRightInput: '#top-right-input',
-  normalDiv: '#noneditable',
-  nonSelectedDiv: '#noneditable-userselectnone'
+  // dialogposition.html
+  DPbottomLeftInput: '#bottom-left-input',
```

Comment:
nit: I would suggest not abbr. it; DialogPosition/Functionality/Noneditable/Bug(?)